materials/internals/definition.yaml

```diff
@@ -0,0 +1,325 @@
+      border-color-input-warning:
+        description: 'bar'
+        deprecated: true
+        choice: color-orange
+      border-color-tag-pristine:
+        description: 'bar'
+        deprecated: true
+        choice: color-gray-60
+      border-color-tag-warning:
+        description: 'bar'
+        deprecated: true
+        choice: color-orange
+      border-color-tag-focus:
+        description: 'bar'
+        deprecated: true
+        choice: color-green
+      border-color-tag-warning-hover:
+        description: 'bar'
+        deprecated: true
+        choice: color-orange
+      border-color-separator:
+        description: 'bar'
+        deprecated: true
+        choice: color-gray
+  borderRadius:
+    label: Border Radius
+    prefix: border-radius
+    decisions:
+      border-radius-input:
+        description: ''
+        deprecated: true
+        choice: border-radius-6
+      border-radius-tag:
+        description: ''
+        deprecated: true
+        choice: border-radius-2
+  # We don't have font families in choices yet, so we can't have decisions for
+  # them
+  # font-family:
+  #   label: Font Family
+  #   decisions:
+  #     default:
+  #       description: ''
+  #       value: '"Open Sans", sans-serif'
+  fontColors:
+    label: Font Colors
+    prefix: font-color
+    decisions:
+      font-color-default:
+        description: ''
+        deprecated: true
+        choice: color-black
+      font-color-disabled:
+        description: ''
+        deprecated: true
+        choice: color-gray-60
+      font-color-error:
+        description: ''
+        deprecated: true
+        choice: color-red
+      font-color-success:
+        description: ''
+        deprecated: true
+        choice: color-green
+      font-color-warning:
+        description: ''
+        deprecated: true
+        choice: color-orange
+      font-color-readonly:
+        description: ''
+        deprecated: true
+        choice: color-gray-60
+      font-color-placeholder:
+        description: ''
+        deprecated: true
+        choice: color-gray-60
+  # We don't have font sizes in choices yet, so we can't have decisions for them
+  # fontSizes:
+  #   label: Font Sizes
+  #   prefix: font-size
+  #   decisions:
+  #     font-size-default:
+  #       description: ''
+  #       value: 13px
+  #     font-size-small:
+  #       description: ''
+  #       value: 12px
+  shadows:
+    label: Shadows
+    prefix: shadow
+    decisions:
+      shadow-box-tag-when-hovered:
+        description: 'foo'
+        deprecated: true
+        choice: shadow-1
+  # There are no choices for these yet so we didn't apply them
+  # size:
+  #   label: Sizes
+  #   decisions: 
+  #     size-height-input:
+  #       description: ''
+  #       value: 32px
+  #     size-height-tag:
+  #       description: ''
+  #       value: 26px
+
+# These tokens are deprecated and not convertible to the new format
+# as we don't have a group for heights
+plainTokens:
```

Comment:
I would have removed them if possible, but they're still in use for the moment. Ultimately we should not use `plainTokens` at all, but it takes a bit to get there.

The current situation is that we use a mix of directly using choices (like `color-green`) and tokens. The tokens we use are old and don't follow the new conventions. They are also not in line with our vision. And they don't even follow the previously established structure (e.g. `  small-button-height` vs `size-height-input` follow completely different naming schemas).

The tokens in use don't follow the `<property>-for-<component-group>-when-<state>` [Token Structure](https://github.com/commercetools/ui-kit/blob/5b27e9beccb63f9d437b5522cc565127d6e44805/philosophy/design-tokens.md#15-token-structure). So all of our current tokens are deprecated (indicated by the `deprecated: true` flag on each decision in `decision.yaml`).

All tokens need to be converted to the new format (by dropping the deprecated token and introducing a new token) and we need to rethink how much sense some of those tokens even make in the first place. This takes some time and requires coordination with the design team. I've been working on this with Sven and we have an idea of where we want to go, which is becoming more and more concrete.

Right now we offer deprecated tokens but we don't offer the "correct" new way yet. The deprecated flag is used to avoid validation of those token names for now and to indicate the deprecated components.